### DISCUSSION TITLE
Semantic Kernel and Autogen

### DISCUSSION THREAD

**Discussion by deepinderdeol** [microsoft/semantic-kernel#9831] [Q&A] [answered]
> Our two teams will be working quickly and diligently to get to this alignment, with the goal of making it available in early 2025. 

Blog post: 
https://devblogs.microsoft.com/semantic-kernel/microsofts-agentic-ai-frameworks-autogen-and-semantic-kernel/

Can you provide some more information on what this means for the current SK agent framework? Are significant changes expected?

**Comment by evchaki** [comment, marked as answer]
@deepinderdeol - this means the SK Agent Framework will continue to collaborate with the AutoGen team to review the most current patterns for multi-agent approaches and bring those stable approaches to the SK Agent Framework when ready.

**Reply by deepinderdeol** [reply]
Ok, thanks
